pom.xml

```diff
@@ -164,6 +164,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-bom</artifactId>
```

Comment:
If there are jars needed at runtime for grpc, then the assembly pom may need to be modified to include those so that they show up in the tarball Not sure what, if anything, needs to be done for this.  Could be a follow on issue.